litmus-portal/frontend/src/components/WelcomeModal/index.tsx

```diff
@@ -30,7 +30,7 @@ const Welcomemodal: React.FC<WelcomemodalProps> = ({ isOpen }) => {
         open={open}
         onClose={handleClose}
         disableBackdropClick
-        disableEscapeKeyDown
+        disableEscapeKeyDown 
```

Comment:
you can remove the unnecessary file changes like this trailing space...don't include these files